examples/json_validation_tests/object-set_RareDisease.json

```diff
@@ -0,0 +1,253 @@
+                    "objectSchemaVersion": "0.0.0",
+                    "objectType": "individual"
+                }
+            },
+            {
+                "objectId": {
+                    "alias": "Unaffected mother",
+                    "centerName": "EBI-TEST"
+                },
+                "minimalPublicAttributes": {
+                    "subjectId": "Unaffected mother",
+                    "biologicalSex": "female",
+                    "phenotypes": [
+                        {
+                            "termId": "NCIT:C94232",
+                            "termLabel": "Unaffected"
+                        }
+                    ]
+                },
+                "organismDescriptor": {
+                    "organismTaxon": {
+                        "termId": "NCBITaxon:9606",
+                        "termLabel": "homo sapiens"
+                    },
+                    "commonName": "human"
+                },
+                "individualRelationships": [
+                    {
+                        "rSource": {
+                            "objectId": {
+                                "alias": "14 year-old boy",
+                                "centerName": "EBI-TEST"
+                            },
+                            "objectType": "individual"
+                        },
+                        "rType": "childOf"
+                    },
+                    {
+                        "rSource": {
+                            "objectId": {
+                                "egaAccession": "EGAB00000000001"
+                            },
+                            "objectType": "submission"
+                        },
+                        "rType": "referencedBy"
+                    }
+                ],
+                "schemaDescriptor": {
+                    "commonSchemaVersion": "0.0.0",
+                    "describedBySchemaUri": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.individual.json",
+                    "objectSchemaVersion": "0.0.0",
+                    "objectType": "individual"
+                }
+            },
+            {
+                "objectId": {
+                    "alias": "Blood sample - 14y Proband",
+                    "centerName": "EBI-TEST"
+                },
+                "objectTitle": "Blood sample of the 14y affected proband",
+                "organismDescriptor": {
+                    "organismTaxon": {
+                        "termId": "NCBITaxon:9606",
+                        "termLabel": "homo sapiens"
+                    },
+                    "commonName": "human"
+                },
+                "sampleStatus": [
+                    {
+                        "caseVsControl": "not applicable",
+                        "conditionUnderStudy": {
+                            "termId": "MONDO:0008029",
+                            "termLabel": "Bethlem myopathy"
```

Comment:
Can instead we use the Orphanet/ORDO ontology for this disease? [Orphanet:610](http://www.orpha.net/ORDO/Orphanet_610) (or ORPHA:610). Drives home the point that we can accept terms from the rare disease ontology.

Reply:
I used [``MONDO:0008029``](http://purl.obolibrary.org/obo/MONDO_0008029) because it was the EFO's import from MONDO. This goes along the ontology validation constraint of both phenotypes and diseases that I placed onto "disease" (see [these lines](https://github.com/EbiEga/ega-metadata-schema/blob/a23839bc7635a35081776cb9d86694d34b7a6b40/schemas/EGA.common-definitions.json#L3803-L3808)).

We could change that rule, perhaps extending it to other ontology hierarchies (e.g. those under ORDO's Disease - [Orphanet:377788](http://www.orpha.net/ORDO/Orphanet_377788)). It's just a matter on agreeing on which ones to use. It is true, though, that we should be wary of terms that refer to the same concept (e.g. ``MONDO:0008029`` and ``Orphanet:377788``) being aggregated downstream. It's true that both should share the same label, but we shouldn't rely on the label, but on the mapping.

Reply:
Given we have rare disease projects we are funded by, we should include these specific ontologies where we can. OK to leave this as-is if we need to do schemas changes to enable the validation against ORDO, and then we can update these examples in the same PR as the schemas change.

Reply:
Roger that. I created **EEH-2506** for this matter.